dspace-api/src/main/java/org/dspace/content/Process.java

```diff
@@ -0,0 +1,204 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content;
+
+import java.util.Date;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.dspace.core.ReloadableEntity;
+import org.dspace.eperson.EPerson;
+
+/**
+ * This class is the DB Entity representation of the Process object to be stored in the Database
+ */
+@Entity
+@Table(name = "process")
+public class Process implements ReloadableEntity<Integer> {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "process_id_seq")
+    @SequenceGenerator(name = "process_id_seq", sequenceName = "process_id_seq", allocationSize = 1)
+    @Column(name = "process_id", unique = true, nullable = false)
+    private Integer processId;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id", nullable = false)
+    private EPerson ePerson;
+
+    @Column(name = "start_time")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date startTime;
+
+    @Column(name = "finished_time")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date finishedTime;
+
+    @Column(name = "script", nullable = false)
+    private String name;
+
+    @Column(name = "status")
+    @Enumerated(EnumType.STRING)
+    private ProcessStatus processStatus;
+
+    @Column(name = "parameters")
+    private String parameters;
+
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(
+        name = "process2bitstream",
+        joinColumns = {@JoinColumn(name = "process_id")},
+        inverseJoinColumns = {@JoinColumn(name = "bitstream_id")}
+    )
+    private List<Bitstream> bitstreams;
+
+    @Column(name = "creation_time", nullable = false)
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date creationTime;
+
+    protected Process() {
+    }
+
+    public Integer getID() {
+        return processId;
+    }
+
+    public void setProcessId(Integer processId) {
+        this.processId = processId;
+    }
+
+    public EPerson getEPerson() {
```

Comment:
Please add Javadocs to all Getters.  In this case we should call out that the Eperson here is the person who kicked off the Process.